components/control_bar.py

```diff
@@ -711,6 +712,23 @@ def create_reset_view_affix():
     )
 
 
+def create_viewfinder_affix():
+    return dmc.Affix(
+        position={"top": 0, "right": 0},
+        zIndex=9999999,
+        children=dmc.Tooltip(
+            label="Toggle viewfinder",
+            position="bottom-start",
+            offset=3,
+            children=dmc.Burger(
```

Comment:
I'm not sure a `Burger` component is intuitive here. I'd vote for a `dmc.Switch`, which has a more clear toggle on/off. I also don't love that the "x" overlaps with the viewfinder. It would be nice to have a switch centered on top of the viewfinder, IMO. 

<img width="135" alt="Screenshot 2023-10-05 at 2 52 34 PM" src="https://github.com/mlexchange/mlex_highres_segmentation/assets/25990216/f6d93739-9b0d-4417-aeb6-ffa469074c61">